addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1552,6 +1556,15 @@ var SnippetsMenu = Widget.extend({
             }
         });
     },
+    /**
+     * @private
+     */
+    _activateTouchEvents() {
+        document.addEventListener("touchstart", this._touchHandler, true);
```

Comment:
Name the handler using our convention `_onXXX`. Here something like `_onTouchEvent`. You should also bind the handler to `this` so the handler can properly be overridden relying on the fact `this` is the Class instance, not something random.